app/app/src/main/java/com/globant/calculator/android/MainActivity.java

```diff
@@ -0,0 +1,23 @@
+package com.globant.calculator.android;
+
+import android.os.Bundle;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.example.myapplication.R;
+import com.globant.calculator.android.mvp.model.CalculatorModel;
+import com.globant.calculator.android.mvp.presenter.CalculatorPresenter;
+import com.globant.calculator.android.mvp.view.CalculatorView;
+
```

Comment:
Here, the same, please remove this black spaces

Reply:
Done, thanks